pkg/controller/deployment/util/deployment_util_test.go

```diff
@@ -1274,13 +1274,19 @@ func TestAnnotationUtils(t *testing.T) {
 
 	//Test Case 1: Check if anotations are copied properly from deployment to RS
 	t.Run("SetNewReplicaSetAnnotations", func(t *testing.T) {
-		//Try to set the increment revision from 1 through 20
-		for i := 0; i < 20; i++ {
+		//Try to set the increment revision from 11 through 20
+		for i := 10; i < 20; i++ {
 
 			nextRevision := fmt.Sprintf("%d", i+1)
-			SetNewReplicaSetAnnotations(&tDeployment, &tRS, nextRevision, true)
+			SetNewReplicaSetAnnotations(&tDeployment, &tRS, nextRevision, true, 5)
 			//Now the ReplicaSets Revision Annotation should be i+1
 
+			if i >= 12 {
```

Comment:
Not sure why it needs to range from 0 and i needs to >= 12? Wouldn't this work when i is 0-11?

Reply:
The first two history annotations are "" and "11"

This condition makes sure the assertion on line 1286 holds.

The change in range of indices is for the length of concatenated history annotations to be regular - both numbers around the comma have two digits.